docs/preview/03-Features/powershell/azure-devops.md

```diff
@@ -18,16 +18,26 @@ PS> Install-Module -Name Arcus.Scripting.DevOps -Repository PSGallery -AllowClob
 
 Assign a value to a DevOps pipeline variable during the execution of this pipeline.
 
-| Parameter       | Mandatory | Description                                       |
-| --------------- | --------- | ------------------------------------------------- |
-| `Name`          | yes       | The name of the variable to set in the pipeline   |
-| `Value`         | yes       | The value of the variable to set in the pipeline  |
+| Parameter             | Mandatory | Description                                       |
+| --------------------- | --------- | ------------------------------------------------- |
+| `Name`                | yes       | The name of the variable to set in the pipeline   |
+| `Value`               | yes       | The value of the variable to set in the pipeline  |
+| `SetVariableAsSecret` | no        | The switch to set the variable as a secret        |
```

Comment:
Would `-AsSecret` be too abstract? Otherwise it's a bit 'duplication' of 'variable' in both function and parameter switch.

Reply:
Fine by me to leave it as this, though.

Reply:
I like `AsSecret` more, so renamed to that 👍🏻 see [`7096e07` (#380)](https://github.com/arcus-azure/arcus.scripting/pull/380/commits/7096e07d7cca0c16a901ba66fcf5fe9f441a83cb)